doc/content/modules/user-manual/pages/release-notes/2025.12.0.adoc

```diff
@@ -12,6 +12,16 @@ image::gv-inherited-port.png[Inherited port, width=65%,height=65%]
 
 == Bug fixes
 
+- Fix the drag and drop action of an `Element` having only a _short name_ and no _declared name_ for the _Explorer View_ to a _General View_.
```

Comment:
Fix an issue where the drag and drop of an `Element` with a _declared short name_ but no _declared name_ from the _Explorer_ view to a diagram was not possible.

Reply:
Done.